source/math/FloatMatrix.ooc

```diff
@@ -123,9 +123,11 @@ FloatMatrix : cover {
 	transpose: func -> This {
 		t := this take()
```

Comment:
I'm kind of new to this take and give thing, but is this something that we use as standard? I mean to use t as a variable name for take. Otherwise I would say it's a bit confusing.